packages/schematics/angular/migrations/update-9/ivy-libraries.ts

```diff
@@ -96,7 +100,9 @@ export function updateLibraries(): Rule {
       }
     }
 
-    tree.commitUpdate(recorder);
+    if (hasUpdate) {
+      tree.commitUpdate(recorder);
```

Comment:
This should really be the inherent behavior of `commitUpdate` but for now this works.

Reply:
Actually you gave me an idea :)